xml/http_servers.xml

```diff
@@ -1907,6 +1907,27 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
+   <fingerprint pattern="^BBC (\d+\.\d+\.\d+); ovbbccb \d+\.\d+\.\d+$">
+      <example>BBC 06.00.083; ovbbccb 06.00.083</example>
+      <example>BBC 06.20.050; ovbbccb 06.20.050</example>
+      <example>BBC 11.10.035; ovbbccb 11.10.035</example>
+      <description>OpenView ovbbccb </description>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.product" value="OpenView ovbbccb"/>
```

Comment:
Sounds good. I can add them. It looks like there are already individual fingerprints for LBB and Coda that don't include version that can be tweaked to fit this format